controllers/consoleplugin/config/static-frontend-config.yaml

```
@@ -913,6 +924,14 @@ scopes:
     feature: multiCluster
     filter: cluster_name
     stepInto: zone
+  - id: udn
+    name: UDN
+    shortName: UDN
+    description: User Defined Network
+    labels:
+      - UdnId
+    filter: udn
+    stepInto: host
```

I think it would be more natural here to step into namespace rather than host. I'm saying that based on the UDN doc: 

> This allows users to be able to design native isolation between their tenant namespaces by coupling all namespaces that belong to the same tenant under the same secondary network and then making this network the primary network for the pod.

(cf https://ovn-kubernetes.io/getting-started/configuration/#enable-network-segmentation)

```suggestion
    stepInto: namespace
```